contracts/partial/yToken/lendingMethods.ligo

```diff
@@ -378,7 +359,7 @@ function borrow(
           s.accounts[(Tezos.sender, yAssetParams.tokenId)] := userAccount;
           s.borrows[Tezos.sender] := borrowTokens;
 
-          const maxBorrowInCU : nat = calcMaxCollateralInCU(
+          const liquidateCollateral : nat = calcLiquidateCollateral(
```

Comment:
This will lead to incorrect calculation.
Should have left it as `calcMaxCollateral`